scene/main/viewport.cpp

```diff
@@ -3848,7 +3848,66 @@ Viewport *Viewport::get_parent_viewport() const {
 
 void Viewport::set_embedding_subwindows(bool p_embed) {
 	ERR_THREAD_GUARD;
-	gui.embed_subwindows_hint = p_embed;
+	if (gui.embed_subwindows_hint == p_embed) {
+		return;
+	}
+
+	bool allow_change = true;
+
+	if (!is_inside_tree()) {
+		// Change can happen, since no child window is displayed.
```

Comment:
```suggestion
		// Change can happen since no child window is displayed.
```

Reply:
Oh my. I have to get better with these. Thanks!